src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,63 @@
+package calculator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class StringCalculatorTest {
+    StringCalculator stringCalculator;
+    private final static String EMPTY_EXPRESSION_MESSAGE = "수식이 비었거나 null 입니다.";
+    private final static String WRONG_EXPRESSION_MESSAGE = "수식이 잘못되었습니다.";
+    private final static String NOT_DIVIDE_BY_ZERO_MESSAGE = "0으로 나눌 수 없습니다.";
```

Comment:
```suggestion
    private final static String EMPTY_EXPRESSION_MESSAGE = "수식이 비었거나 null 입니다.";
    private final static String WRONG_EXPRESSION_MESSAGE = "수식이 잘못되었습니다.";
    private final static String NOT_DIVIDE_BY_ZERO_MESSAGE = "0으로 나눌 수 없습니다.";
    
    private StringCalculator stringCalculator;
```

테스트코드에서도 컨벤션을 지켜주면 좋겠어요 :)

Reply:
네 알겠습니다!